lib/routemap.h

```diff
@@ -153,6 +153,7 @@ struct route_map_index {
 
 	/* Keep track how many times we've try to apply */
 	uint64_t applied;
+	uint64_t applied_clear;
```

Comment:
Question: why do we need this new variable? Couldn't the new command just set `applied` to zero?

Reply:
what if in the future someone wants to know the total irrelevant to cleared?  That is the reason why I did it this way

Reply:
Ok, thanks for the explanation.